### PR TITLE
improve logging and ask user to provide --gcp-project for kubetest local run

### PR DESCRIPTION
**Pull request by krzyzacy**
/assign @rmmh @cjcullen

### CODE REVIEW

File: kubetest/main.go [hunk 1]
```diff
@@ -717,6 +717,9 @@ func prepareGcp(o *options) error {
 		os.Setenv("NODE_OS_DISTRIBUTION", nod)
 	}
 	if o.gcpProject == "" {
+		log.Print(`--gcp-project is missing, try to fetch a project from boskos.
+				  (for local run please point --gcp-project to your dev project)`)
```
Comment (rmmh) [inline kubetest/main.go]: "trying to fetch", and "for local runs set --gcp-project to your dev project"

Comment (krzyzacy) [reply kubetest/main.go]: done

File: kubetest/main.go [hunk 2]
```diff
@@ -717,6 +717,9 @@ func prepareGcp(o *options) error {
 		os.Setenv("NODE_OS_DISTRIBUTION", nod)
 	}
 	if o.gcpProject == "" {
+		log.Print(`--gcp-project is missing, trying to fetch a project from boskos.
+				  (for local runs please set --gcp-project to your dev project)`)
```
Comment (rmmh) [inline kubetest/main.go]: won't this print a newline and bunch of whitespace?

Comment (krzyzacy) [reply kubetest/main.go]: hummm gofmt allows that

**Pull request merged**